packages/browser-utils/src/metrics/browserMetrics.ts

```diff
@@ -300,6 +300,13 @@ interface AddPerformanceEntriesOptions {
    * sent as a standalone span instead.
    */
   recordClsOnPageloadSpan: boolean;
+
+  /**
+   * Resource spans matching strings in the array will not be emitted.
+   *
+   * Default: []
+   */
+  ignoreResourceSpans: Array<string>;
```

Comment:
I'm thinking we should type out well-known ops so that users get some suggestions in their IDEs when setting this option. We can still allow `string` as well so that any resources we don't explicitly name are ignored, too. 

```suggestion
  ignoreResourceSpans: Array<'resouce.script' | 'resource.css' | 'resource.img' | 'resource.other' | string >;
```